src/redux.js

```diff
@@ -602,7 +605,8 @@ export default function rootReducer(state = initialState, action) {
   if (action.type === SET_SAVE_STATUS) {
     return {
       ...state,
-      saveStatus: action.status
+      saveStatus: action.status,
+      saveResponseData: action.data
```

Comment:
Probably out of scope: Would it be worth breaking this into multiple redux files? This file's getting pretty big.

Reply:
It could be interesting, but definitely out of scope for this change.